benchmark/sparse/linear_regression/inference_benchmark.md

```diff
@@ -0,0 +1,113 @@
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 64.2 sec | 64.3 sec
+| C5.8XLarge |   0  | 128 | 32.1 sec | 31.4 sec 
+| C5.8XLarge |   0  | 256 | 16.5 sec | 16.3 sec
+| C5.8XLarge |   0  | 512 | 8.3 sec | 8.8 sec 
+| C5.8XLarge |   0  | 1024 | 4.3 sec | 4.4 sec
+
+#### Memory utilization
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (Mem %) | Keras-TensorFlow (Mem %)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 0.4 | 0.5 |
+| C5.8XLarge |   0  | 128 | 0.4 | 0.5 | 
+| C5.8XLarge |   0  | 256 | 0.4 | 0.5  |
+| C5.8XLarge |   0  | 512 | 0.4 | 0.5 |
+| C5.8XLarge |   0  | 1024 | 0.4 | 0.5 |
+
+#### Memory consumed
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (MB) | Keras-TensorFlow (MB)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 1630.8 | 1573.8 |
+| C5.8XLarge |   0  | 128 | 1574.7 | 1561.2 | 
+| C5.8XLarge |   0  | 256 | 1477.8 | 1501.4  |
+| C5.8XLarge |   0  | 512 | 1407.0| 1472.5 |
+| C5.8XLarge |   0  | 1024 | 1336.3 | 1466.8 |
+
+#### GPU
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.2                                                      |
+| TensorFlow-GPU   | v1.11.0                                                      |
+| MXNet-cu90mkl    | v1.3.0   
+
+###### Using 10 epochs
+##### Single GPU
+##### Speed
+###### Note speed calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   1  | 64  | 3.0 sec | 2.0 sec
+| P3.8XLarge |   1  | 128 | 2.3 sec | 1.2 sec 
+| P3.8XLarge |   1  | 256 | 1.2 sec | 0.7 sec
+| P3.8XLarge |   1  | 512 | 0.8 sec | 0.5 sec
+| P3.8XLarge |   1  | 1024 | 0.4 sec | 0.4 sec
+
+##### Memory utilization
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (GPU Utilization %) | Keras-TensorFlow (GPU Utilization %)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   1  | 64  | 9 | 6
+| P3.8XLarge |   1  | 128 | 8 | 7
+| P3.8XLarge |   1  | 256 | 8 | 7
+| P3.8XLarge |   1  | 512 | 8 | 7
+| P3.8XLarge |   1  | 1024 | 7 | 8
+
+##### Memory consumed
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (MB) | Keras-TensorFlow (MB)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   1  | 64  | 966.7 | 16135.5
+| P3.8XLarge |   1  | 128 | 970.9 | 16135.5
+| P3.8XLarge |   1  | 256 | 973.1 | 16135.5
+| P3.8XLarge |   1  | 512 | 994.1 | 16135.5
+| P3.8XLarge |   1  | 1024 | 987.7 | 16135.5
+
+##### Multi-GPU
+##### Speed
+###### Benchmark results on multi GPU calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   2  | 512 | 0.86 sec | Not supported
```

Comment:
We generally don't use multi-gpu for inference, and here the inference speed is actually slower than 1 GPU for batch 512 and 1024. Please verify or remove this section.

Reply:
Removed